ext/CEGUI/IrrlichtBaWTexture.cpp

```diff
@@ -77,6 +142,9 @@ const CEGUI::Vector2f& IrrlichtBaWTexture::getTexelScaling() const
 
 void IrrlichtBaWTexture::loadFromFile(const CEGUI::String& filename, const CEGUI::String& resourceGroup)
 {
+    if (frameBuffer)
+        frameBuffer->drop();
```

Comment:
why are you turning every single texture into a framebuffer!?

Reply:
adjusted in following commit